examples/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 3.11)
+
+get_filename_component(ImGui_SRCDIR "${CMAKE_CURRENT_LIST_DIR}" DIRECTORY)
```

Comment:
The mix of CamelCase and ALL_CAPS is a bit unreadable. Isn't it better to go full ALLCAPS? IMGUI_SRCDIR looks a bit better, I think.

Reply:
Agreed.

Reply:
Who else wants to use the ALL_CAPS convention for variables?

Reply:
I would also suggest going ALL_CAPS for the IMGUI_XXX variable, unless you thing the current way is generally more standard in cmake land (the `ImGui_OPTIONS_XXX` stuff looks odd).